src/components/Checkbox/styles.ts

```diff
@@ -66,7 +66,8 @@ export const StyledLabel = styled.label<StyledLabelProps>`
       : theme.colors.inputBorder};
   cursor: pointer;
   display: block;
-  font-weight: ${({ theme, checked }) => (checked || !theme.utilities.useDefaultFocusRec ? 'bold' : 'normal')};
+  font-weight: ${({ theme, checked }) =>
+    checked || !theme.utilities.useDefaultFocusRect ? theme.typography.fontWeightHeavy : 'normal'};
```

Comment:
I probably should have been more explicit than normal and used "fontWeightMedium"